api/contract.go

```diff
@@ -186,11 +186,12 @@ type updateFilterDimensionResponse struct {
 }
 
 type dimensionItem struct {
-	ID             string             `json:"id"`
-	Name           string             `json:"name"`
-	Label          string             `json:"label"`
-	FilterByParent string             `json:"filter_by_parent,omitempty"`
-	Links          dimensionItemLinks `json:"links"`
+	ID                    string             `json:"id"`
+	Name                  string             `json:"name"`
+	Label                 string             `json:"label"`
+	FilterByParent        string             `json:"filter_by_parent,omitempty"`
+	DefaultCategorisation string             `json:"default_categorisaion"`
```

Comment:
Would we want to omitempty?

Reply:
i dont think so based on the conversation me and Jit had